pkg/skaffold/inspect/jobManifestPaths/modify.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package inspect
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+
+	"github.com/GoogleContainerTools/skaffold/v2/pkg/skaffold/config"
+	"github.com/GoogleContainerTools/skaffold/v2/pkg/skaffold/parser"
+	"github.com/GoogleContainerTools/skaffold/v2/pkg/skaffold/schema/latest"
+	"github.com/GoogleContainerTools/skaffold/v2/pkg/skaffold/util"
+	"github.com/GoogleContainerTools/skaffold/v2/pkg/skaffold/yaml"
+)
+
+var (
+	getCfgs = parser.GetAllConfigs
+)
+
+func Modify(ctx context.Context, out io.Writer, opts config.SkaffoldOptions, inputFile, outputFile string) error {
+	// Open our jsonFile
+	jsonFile, err := os.Open(inputFile)
+	if err != nil {
+		fmt.Println(err)
+	}
+	// defer the closing of our jsonFile so that we can parse it later on
+	defer jsonFile.Close()
+
+	byteValue, _ := ioutil.ReadAll(jsonFile)
+
+	var result jobManifestPathList
+	json.Unmarshal(byteValue, &result)
```

Comment:
This operation can return an error, if the `inputFile` is not a json file. I noticed that we fail silently, and the generated output is the same skaffold.yaml without the modifications. Should we also log the error from the unmarshal?

Reply:
Yep, good catch.  This now properly handles errors and has a test case related to this

Reply:
Also logs the error as well to stdout for users to know what happened